CHANGES.md

```diff
@@ -1,3 +1,37 @@
+# 2.16.0 - 29 Aug 2024
+
+## Features
+
+* Parse JUJU_* env in one place in https://github.com/canonical/operator/pull/1313
```

Comment:
```suggestion
* Parse JUJU_* environment variables in one place in https://github.com/canonical/operator/pull/1313
```

I probably should have asked this in the actual PR, since it probably starts with `feat:` but I wonder if this is really a feature? We are partly doing it to make it easier for people to experiment with ops, so that is at least feature-adjacent, but mostly it's internal, so it feels like it probably should be in the "Refactor" section. What do you think?